### DISCUSSION TITLE
Clear logs??

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#2364] [General]
Silly question... My disk space is filling up and I need to clear my logs. How do I go about this?

**Comment by greatapoc** [comment]
so-elastic-clear will remove everything.

**Reply by angcar** [reply]
Hi,
And this does not affect the shards of elasticsearch?

Thanks.

**Comment by ColeVan** [comment]
so-elastic-clear does not seem to work. I have tried it multiple times. I even rebooted it twice. Logs are still there. Is there a config that can be modified to clear logs over a certain age? For example, logs over 30 days will be deleted. any idea why so-elastic-clear isn't working?
I am running a distributed-environment using Version 2.3.1
Thanks! 
![image](https://user-images.githubusercontent.com/70167373/102478406-8c817d80-402b-11eb-9cbf-1aa7b8817eed.png)
![image](https://user-images.githubusercontent.com/70167373/102478468-9efbb700-402b-11eb-9cf7-78a78af808aa.png)

**Reply by kunwon1** [reply]
From that screenshot, it looks like you didn't type AGREE and press enter - I did this yesterday and it worked

**Reply by ColeVan** [reply]
Well..... I feel stupid lol. Thanks. It worked

**Comment by TOoSmOotH** [comment]
Are you collecting PCAP?

**Reply by ColeVan** [reply]
Yes

**Reply by TOoSmOotH** [reply]
Then that is what is filling up your disk. You can disable it if you have space issues.
